### PR TITLE
Add support for an apiVersion option

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Implement the #132 requested feature

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -11,7 +11,11 @@
     <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
             <div class="navbar-header">
-                <a class="navbar-brand" href="#">Route Documentation</a>
+                <a class="navbar-brand" href="#">
```
Comment (Marsup) [inline templates/index.html]: Keep it on a single line please. Also I doubt that's the correct syntax.

**Comment by Marsup** [comment]
Missing tests.

**Comment by AdrieanKhisbe** [comment]
voilà :)

**Comment by AdrieanKhisbe** [comment]
Test non passing because of 0.10:

      /home/travis/build/hapijs/lout/node_modules/hapi/lib/index.js:5
     const Server = require('./server');
     ^^^^^
     SyntaxError: Use of const in strict mode.

shall I update travis test version, to only target 4 and +?

File: templates/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
             <div class="navbar-header">
-                <a class="navbar-brand" href="#">Route Documentation</a>
+                <a class="navbar-brand" href="#">Route Documentation{{#@apiVersion}} - v{{@apiVersion}}{{/@apiVersion}}</a>
```
Comment (Marsup) [inline templates/index.html]: Where is that #@ form documented ? handlebars seems to lack any documentation about that.

**Comment by Marsup** [comment]
Yes go for it.

File: templates/index.html (hunk 2 above)
Comment (AdrieanKhisbe) [inline templates/index.html]: on SO ^^
http://stackoverflow.com/questions/6027525/how-do-i-accomplish-an-if-else-in-mustache-js

**Comment by AdrieanKhisbe** [comment]
Do you want me to set a coveralls badge?

(and update the dependancies, cf: https://david-dm.org/hapijs/lout#info=devDependencies)

File: .travis.yml [hunk 3]
```diff
@@ -4,8 +4,13 @@ sudo: false
 language: node_js
 
 node_js:
-  - "0.10"
-  - "4.0"
+  - 4.0
```
Comment (Marsup) [inline .travis.yml]: Stick to quotes, and keep "4" "4.0" "5".
No cache.

File: templates/index.html (hunk 2 above)
Comment (Marsup) [inline templates/index.html]: We're using handlebars, not mustache, I'd rather not rely on undocumented features, so use a if.

Comment (AdrieanKhisbe) [inline templates/index.html]: My bad, I had mixed up the two

File: .travis.yml [hunk 4]
```diff
@@ -4,8 +4,10 @@ sudo: false
 language: node_js
 
 node_js:
-  - "0.10"
   - "4.0"
+  - "4.1"
```
Comment (Marsup) [inline .travis.yml]: Still not good, I gave the exact versions that need to appear.

Comment (Marsup) [inline .travis.yml]: Can you set that on appveyor as well ?

Comment (AdrieanKhisbe) [inline .travis.yml]: Yep, gave me the exact version you want, 
and I'll configure for both :)

Comment (Marsup) [inline .travis.yml]: I already did a few comments back.

Comment (AdrieanKhisbe) [inline .travis.yml]: confirm me you don't want to test against 4.2.2. LTS, and latest 5.1?
I'll only do 4.1 otherwise

Comment (Marsup) [inline .travis.yml]: I want exactly "4" "4.0" "5", nothing else.

**Comment by Marsup** [comment]
Merged manually in 608a80f to make a few fixes, thanks.

**Pull request closed without merging**